website/docs/integrations/services/opnsense/index.md

```diff
@@ -11,26 +11,30 @@ OPNsense is a free and Open-Source FreeBSD-based firewall and routing software.
 :::
 
 :::note
-This is based on authentik 2021.10.3 and OPNsense 21.7.4-amd64 installed using https://docs.opnsense.org/manual/install.html. Instructions may differ between versions.
+This is based on Authentik 2021.10.3 and OPNsense 21.7.4-amd64 installed using https://docs.opnsense.org/manual/install.html. Instructions may differ between versions.
```

Comment:
Also I know this is very nitpicky, but lowercase authentik is actually the correct name 😛

Reply:
Damn, I totally get that. I'm just the other way, always wanting it to start with an uppercase XD. I'll fix that for ya now